packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -69,19 +69,34 @@ function EnableNotification({
 
   const hasEnabled = (isSubscribed || hasPermissionCache) && isEnabled;
 
+  const conditions = [
+    !isLoaded,
+    isDismissed,
+    !isInitialized,
+    !isNotificationSupported,
+    (isSubscribed || hasPermissionCache) && !isEnabled,
+  ];
+  const shouldNotDisplay = conditions.some((passed) => passed);
+
+  useEffect(() => {
+    if (shouldNotDisplay) {
+      return;
+    }
+
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_id: source,
```

Comment:
Can you please add the target id to the click event as well please?

Reply:
On click event I believe it was also included but was applied on the `origin` property. Should I move the placements below:
https://github.com/dailydotdev/apps/blob/3a66e6938c1d3e7478ffb083a08c2d938308e893/packages/shared/src/hooks/useEnableNotification.tsx#L29-L32

Reply:
Oh then let's do the same for the impression

Reply:
Cool. I will send them a message too, just so they will be aware.

Reply:
Let me know when you change it so I can approve